config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  get 'entities/index'
+  get 'entities/show'
+  get 'entities/new'
+  get 'entities/create'
+  get 'entities/update'
+  get 'entities/destroy'
+  get 'entities/edit'
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root to: 'users#index'
```

Comment:
- [ ] Your root path seems to be different for the deployed and local versions. From the requirements your root path should point to the categories list whereas your local version points to the users index. Kindly ensure your app is working properly

  local version root route | Deployed version root path
  :---:|:---:
  ![fhju](https://user-images.githubusercontent.com/54848363/195971912-725c93b8-ad9f-4b2d-950c-921914ce020f.jpg) | ![dmk](https://user-images.githubusercontent.com/54848363/195971910-5d341623-0628-4180-9524-9e7048e596dc.jpg)